public/static/docs/command-reference/import.md

```diff
@@ -1,10 +1,10 @@
 # import
 
-Download a file or directory from any <abbr>DVC repository</abbr> (e.g. hosted
-on GitHub) into the <abbr>workspace</abbr>. This also creates a
-[DVC-file](/doc/user-guide/dvc-file-format) with information about the data
-source, which can later be used to [update](/doc/command-reference/update) the
-import.
+Download a file or directory from any <abbr>Git repository</abbr> or <abbr>DVC
+repository</abbr> (e.g. hosted on GitHub) into the <abbr>workspace</abbr>. This
```

Comment:
Let's please use

`Download a file or directory from any <abbr>DVC project</abbr> or Git repository into the <abbr>workspace</abbr>.`

To match https://github.com/iterative/dvc/pull/3097/files#diff-25975cd8d9e267641bd09af2c8cb32d5

Reply:
Done!